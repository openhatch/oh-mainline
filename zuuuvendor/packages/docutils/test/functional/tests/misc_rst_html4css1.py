# Source and destination file names.
test_source = "link_in_substitution.txt"
test_destination = "misc_rst_html4css1.html"

# Keyword parameters passed to publish_file.
reader_name = "standalone"
parser_name = "rst"
writer_name = "html4css1"

# Settings
# test for encoded attribute value:
settings_overrides['stylesheet'] = 'foo&bar.css'
settings_overrides['stylesheet_path'] = ''
settings_overrides['embed_stylesheet'] = False
