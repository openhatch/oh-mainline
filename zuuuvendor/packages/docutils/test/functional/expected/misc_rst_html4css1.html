<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
<title></title>
<link rel="stylesheet" href="foo&amp;bar.css" type="text/css" />
</head>
<body>
<div class="document">


<p>Test the interaction of transforms.references.Substitutions and
transforms.references.ExternalLinks.</p>
<p><a class="reference external" href="http://docutils.sourceforge.net/rst.html">reStructuredText</a> is cool!</p>
<p>There is a preferred alternative:</p>
<p><a class="reference external" href="http://docutils.sourceforge.net/rst.html">reStructuredText</a> is cool!</p>
<p>This only works for the case where the entire substitution is a
reference, not when the reference is embedded as part of the
substitution:</p>
<p>unstructured <a class="reference external" href="http://docutils.sourceforge.net/rst.html">reStructuredText</a> is uncool!</p>
</div>
</body>
</html>
